Client/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Paged/BraveSkusScriptHandler.swift

```diff
@@ -70,87 +75,27 @@ class BraveSkusScriptHandler: TabContentScript {
     switch methodId {
     case Method.refreshOrder.rawValue:
       if let orderId = data["orderId"] as? String {
-        handleRefreshOrder(for: orderId, domain: requestHost, replyHandler: replyHandler)
+        
+        braveSkusManager.refreshOrder(for: orderId, domain: requestHost) { result in
+          replyHandler(result, nil)
+        }
       }
     case Method.fetchOrderCredentials.rawValue:
       if let orderId = data["orderId"] as? String {
-        handleFetchOrderCredentials(for: orderId, domain: requestHost, replyHandler: replyHandler)
+        braveSkusManager.fetchOrderCredentials(for: orderId, domain: requestHost) { result in
+          replyHandler(result, nil)
+        }
       }
     case Method.prepareCredentialsPresentation.rawValue:
-      if let domain = data["domain"] as? String, let path = data["path"] as? String {
-        handlePrepareCredentialsSummary(for: domain, path: path, replyHandler: replyHandler)
-      }
+      assertionFailure("The website should never call the credentialsPresentation.")
```

Comment:
`prepareCredentialsPresentation` can never be called from the website anymore, added assertion here